src/components/spreadsheet/utils/config-tables.js

```diff
@@ -103,6 +103,90 @@ export const TABLES_DEFINITIONS = {
                 filter: 'agNumberColumnFilter',
                 fractionDigits: 0,
             },
+            {
+                id: 'LowVoltageLimitkV',
+                field: 'lowVoltageLimit',
+                numeric: true,
+                filter: 'agNumberColumnFilter',
+                fractionDigits: 1,
+                changeCmd: 'equipment.setLowVoltageLimit({})\n',
+                editable: true,
+                cellEditor: NumericalField,
+                cellEditorParams: (params) => {
+                    return {
+                        maxExpression: 'highVoltageLimit',
+                        defaultValue: params.data.lowVoltageLimit,
+                        gridContext: params.context,
+                        gridApi: params.api,
+                        data: params.data,
+                        colDef: params.colDef,
+                    };
+                },
+                getQuickFilterText: excludeFromGlobalFilter,
+            },
```

Comment:
can you refactor this in function and passing parameters? because there is a redundant bloc

Reply:
Done (new function added to generate the new column definitions)